### PR TITLE
Proto fix

### PR DESCRIPTION
**Pull request by ajax-torbin-v**
Added status as enum instead of string

### CODE REVIEW

File: domainservice/src/main/kotlin/com/example/delivery/mapper/OrderProtoMapper.kt [hunk 1]
```diff
@@ -46,7 +46,7 @@ object OrderProtoMapper {
         return FindOrderByIdResponse.newBuilder().apply {
             successBuilder.orderBuilder.also {
                 it.id = id
-                it.status = status
+                it.status = Order.Status.valueOf("STATUS_$status")
```
Comment (ajax-yurkovskyi-t) [inline domainservice/src/main/kotlin/com/example/delivery/mapper/OrderProtoMapper.kt]: provide fallback for UNSPECIFIED value

**Review by ajax-yurkovskyi-t — COMMENTED** [review]
Good job

**Review by ajax-malov-n — COMMENTED** [review]
Ok 🚀

**Review by ajax-yurkovskyi-t — APPROVED** [review]
nice

**Review by ajax-shulepa-a — APPROVED** [review]

**Pull request merged**